configure.ac

```diff
@@ -202,6 +202,19 @@ AC_CHECK_HEADERS(fcntl.h unistd.h bfd.h)
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
 
+HAVE_BFD_XTENSA_PDIFF_RELOCS=0
+case $target in
+	xtensa*)
+		OLD_CFLAGS="$CFLAGS"
```

Comment:
use `AS_VAR_COPY([OLD_CPPFLAGS], [CPPFLAGS])`

`AS_VAR_COPY` has been around for more than a decade now

Reply:
Done